src/main/java/our/yurivongella/instagramclone/service/CommentService.java

```diff
@@ -0,0 +1,141 @@
+    }
+
+    @Transactional
+    public CommentResponseDto createComment(Long postId, CommentCreateDto commentCreateDto) {
+        Long currentMemberId = SecurityUtil.getCurrentMemberId();
+        Member member = memberRepository.findById(currentMemberId)
+                                        .orElseThrow(() -> new UsernameNotFoundException("유저가 존재하지 않습니다."));
+
+        Post post = postRepository.findById(postId)
+                                  .orElseThrow(() -> new NoSuchElementException("해당 게시물이 없습니다."));
+
+        Comment comment = new Comment();
+        comment.create(member, post, commentCreateDto.getContent());
+        commentRepository.save(comment);
+        return CommentResponseDto.of(comment, member);
+    }
+
+    @Transactional
+    public ProcessStatus deleteComment(Long commentId) throws Exception {
+        Long currentMemberId = SecurityUtil.getCurrentMemberId();
+        Member member = memberRepository.findById(currentMemberId)
+                                        .orElseThrow(() -> new UsernameNotFoundException("유저가 존재하지 않습니다."));
+
+        Comment comment = commentRepository.findById(commentId)
+                                           .orElseThrow(() -> new NotFoundException("존재하지 않는 댓글입니다."));
+
+        if (!member.getId().equals(comment.getMember().getId())) {
+            log.error("유저가 일치하지 않습니다.");
+            return ProcessStatus.FAIL;
+        }
+
+        try {
+            commentRepository.deleteById(commentId);
+            return ProcessStatus.SUCCESS;
+        } catch (Exception e) {
+            log.error("삭제 도중 문제가 발생했습니다 = {}", e.getMessage());
+        }
+        return ProcessStatus.FAIL;
+    }
+
+    @Transactional
+    public ProcessStatus likeComment(Long commentId) throws Exception {
+        Long currentMemberId = SecurityUtil.getCurrentMemberId();
+        Member member = memberRepository.findById(currentMemberId)
+                                        .orElseThrow(() -> new UsernameNotFoundException("유저가 존재하지 않습니다."));
+
+        Comment comment = commentRepository.findById(commentId)
+                                           .orElseThrow(() -> new NoSuchElementException("댓글이 존재하지 않습니다."));
+
+        boolean check = commentLikeRepository.findAllByCommentId(commentId)
+                                             .stream()
+                                             .anyMatch(cl -> cl.getMember().getId().equals(member.getId()));
+        if(!check){
+            log.info("{}가 댓글 {}를 좋아요 표시합니다.", member.getName(), comment.getContent());
+            CommentLike commentLike = createCommentLike(member, comment);
+            commentLikeRepository.save(commentLike);
+            return ProcessStatus.SUCCESS;
+        }
+        return ProcessStatus.FAIL;
+    }
+
+    private CommentLike createCommentLike(Member member, Comment comment) {
+        CommentLike commentLike = new CommentLike();
+        commentLike.like(member, comment);
+        return commentLike;
+    }
```

Comment:
```java
public CommentLike like(Member member, Comment comment) {
    this.member = member;
    this.comment = comment;
    comment.getCommentLikes().add(this);
    return this;
}
```

`like` 메소드가 `CommentLike` 를 반환하게 되어있는데 실제로 사용되고 있지 않아요.
한 줄로 바꿀수 있을것같아요

```suggestion
    private CommentLike createCommentLike(Member member, Comment comment) {
        return new CommentLike().like(member, comment);
    }
```

Reply:
[1735a71](https://github.com/INSPJT/instagram_clone_backend/pull/33/commits/1735a716e587b7b4261062667410f6d9af7b1498) 굿굿!